tests/tests_ut/az_ulib_ipc_ut/az_ulib_ipc_ut.c

```diff
@@ -291,15 +425,17 @@ static void az_ulib_ipc_try_get_interface_with_ipc_not_initialized_failed(void**
   /// act
   /// assert
   AZ_ULIB_ASSERT_PRECONDITION_CHECKED(az_ulib_ipc_try_get_interface(
-      AZ_SPAN_FROM_STR(MY_INTERFACE_1_123_INTERFACE_NAME),
-      MY_INTERFACE_1_123_INTERFACE_VERSION,
-      AZ_ULIB_VERSION_EQUALS_TO,
+      AZ_SPAN_EMPTY,
+      AZ_SPAN_FROM_STR(MY_PACKAGE_A_NAME),
+      MY_PACKAGE_1_VERSION,
+      AZ_SPAN_FROM_STR(MY_INTERFACE_1_NAME),
+      MY_INTERFACE_123_VERSION,
       &interface_handle));
 
   /// cleanup
 }
 
-/* If the provided interface name is NULL, the az_ulib_ipc_try_get_capability shall fail with
+/* If the provided interface handle is NULL, the az_ulib_ipc_try_get_capability shall fail with
  * precondition. */
 static void az_ulib_ipc_try_get_capability_with_null_name_failed(void** state)
```

Comment:
>az_ulib_ipc_try_get_capability_with_null_name_failed [](http://example.com/codeflow?start=12&length=52)

should change name of test to reflect what's being tested:
`az_ulib_ipc_try_get_capability_with_null_handle_failed(void** state)`

Reply:
Agree, good catch.